### PR TITLE
fix(ru): translation for a 'statement' term

### PR DESCRIPTION
**Pull request by devope**
Statement in russian translates as a "инструкция".

### CODE REVIEW

**Comment by sashasushko** [comment]
@devope sorry for long response. Could you please change translation in whole article?

**Comment by devope** [comment]
@sashasushko no problem! 
done.

**Comment by sashasushko** [comment]
@devope thanks a lot 🙏 I'll review your PR right after I finished with markdown conversion

**Comment by queengooborg** [comment]
Since this PR has been sitting around for a while, I've gone ahead and made the appropriate changes to effectively rebase this onto the Markdown version. This should now be ready to review!

**Review by sashasushko — APPROVED** [review]

File: files/ru/web/javascript/reference/statements/label/index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ label :
 - `label`
   - : Идентификатор. Любое слово, кроме ключевых и зарезервированных в JavaScript.
 - `statement`
-  - : Выражение. С `break` можно использоваться любое блочное выражение, а с `continue` только циклы.
+  - : Инструкция.`break` может быть использован внутри любого блока инструкций с меткой, а с `continue` только внутри циклов.
```
Comment (sashasushko) [inline files/ru/web/javascript/reference/statements/label/index.md:22]:
```suggestion
  - : Инструкция. `break` может быть использован внутри любого блока инструкций с меткой, а с `continue` только внутри циклов.
```

**Pull request merged**